api/app/signals/apps/api/v1/urls.py

```diff
@@ -42,6 +42,12 @@
     basename='feedback-forms'
 )
 
+signal_router_v1.register(
+    r'user/filters',
```

Comment:
This is still TBD.

`/signals/v1/me/filters/`
`/signals/v1/private/me/filters/`
...

Reply:
Same as where we let the model live. Let's talk about it.